app/views/posts/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="post-section">
+  <% content_for :title, "#{@user.name}'s Posts" %>
+  <%= render "users/user", user: @user %>
+  <% @user.posts.each do | post | %>
+    <%= link_to user_post_path(@user, post) do %>
+      <%= render "post", post: post %>
+
+      <div>
+        <h4>
+          <span>Comments</span>
+          <span>for Post: </span>
+          <span><%="#{post.title}"%> </span>
+        </h4>
+        <div class='postComments'>
+          <% post.recent_comments.each do |comment| %>
+            <div>
+              <p>
+                <span><strong><%= comment.user.name%>:</strong></span>
+                <span><%= comment.text %></span>
+              </p>
+            </div>
+            <%if can? :destroy, comment%>
+              <%= button_to "Delete Comment",
+               user_post_comment(comment.author_id,
```

Comment:
- [ ] You mentioned, to the previous reviewer, that the delete comment feature has been implemented in the posts index page, but when I visit the route, it presents me with an error pertaining to this path. Kindly fix that and ensure thet comments can actually be deleted.

  ![image](https://github.com/Fahim-Khan-P/blog/assets/36542986/76ce720d-5b1a-4997-a97b-48768addc3b7)